.travis.yml

```diff
@@ -64,11 +81,21 @@ install:
   - apt-cache policy docker-ce
   - sudo apt-get install -y docker-ce
   - sudo usermod -aG docker ${USER}
-  - if [ "$ANSIBLE_VERSION" = "latest" ]; then pip install ansible[azure]; else pip
-    install ansible[azure]==$ANSIBLE_VERSION; fi
+#  - if [ "$ANSIBLE_VERSION" = "latest" ]; then pip install ansible[azure]; else pip
+#    install ansible[azure]==$ANSIBLE_VERSION; fi
+#  - pip install ansible[azure]==2.6.0.rc3
+#  - pip install ansible[azure]==2.5.0
+  - git clone https://github.com/ansible/ansible.git
+  - pip install -r ./ansible/requirements.txt
+  - pip install -r ./ansible/packaging/requirements/requirements-azure.txt 
+  - cd ansible
+  - source ./hacking/env-setup
```

Comment:
Thanks Zim for  the great effort to test different ansible release with sample playbooks!

I think we dont' need to cover devel branch in our sample repo. samples should test against official ansible release, not devel branch. sample repo ci is used to make sure samples itself works with released ansible, not to test un-release ansible to find out ansible bugs. unrelease ansible should be covered by upstream ci or vschina/ansible ci.

Reply:
this is a starting point, and devel branch is the only one where i can make all the tests working. later will be replaced with proper branches when we fix all the issues

Reply:
so in  future, you'll still use unrelease branch to test sample repo? like stable-2.5? i think ansible in sample ci should be installed by pip install, not from ansible branch.

to test unrelease ansible branch, 2 things we can do:
- backport process.
- ci in vschina/ansible on stable-2.5, stable-2.6 branch.

Reply:
no, at the moment i will:
- use branches
- backport fixes to these branches
- when there's minor release with fixes, i will switch to the release instead of branch